examples/Example/src/new_api/chat_heads/index.tsx

```diff
@@ -0,0 +1,183 @@
+import React, { useState } from 'react';
+import { StyleSheet, ImageStyle, LayoutChangeEvent } from 'react-native';
+import { Gesture, GestureDetector } from 'react-native-gesture-handler';
+import Animated, {
+  useAnimatedStyle,
+  useDerivedValue,
+  useSharedValue,
+  withSpring,
+} from 'react-native-reanimated';
+import { SafeAreaView } from 'react-native-safe-area-context';
+
+const HEADER_HEIGHT = 44; // let's assume it's always the same for ease of understanding ;)
```

Comment:
Can't we get it from [navigation](https://reactnavigation.org/docs/elements/#useheaderheight)?

Reply:
It wasn't not part of this PR. Removed this file.